### ISSUE TITLE
Not able to press caps+control+down arrow to change setting

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5169]
**Reported by masonian99 on 2015-06-20 10:21**
When I try to press control+NVDA+down arrow on synths settings ring, it will not le change the item. Furthermore, I change the hotkey and it still doesn't work.

**Comment by ehollig** [comment]
This works as expected for me when I enable the desktop layout. If it is not possible to contact the original author, I would say this can be closed.

**Comment by leonardder** [comment]
Agreed.
